### PR TITLE
Digic 6 support, 750D, 200D, R, M50 into LV, scripts and fixes

### PR DESCRIPTION
**Pull request by kitor**
* Added support for full build of DIGIC6 cameras
* Dialog structs for D6, D7 and EOSR (not "D8" yet as M/RP not tested, RP is known to differ in size)
* New cam supported: 750D - features on par with 200D,R
* 750D, 200D, R, M50 have working Global draw in LV + overlays feature
* R: Experimental support for CONFIG_RESTORE_AFTER_FORMAT
* Fixes in rbf_font, propvalues
* Scripts for decoding D8 palette resources and D7/D8 EDMAC Config

### CODE REVIEW

**Comment by reticulatedpines** [comment]
Taken with tweaks into d6_plus_d678_lv

File: src/lvinfo.c [hunk 1]
```diff
@@ -105,7 +105,8 @@ static int lvinfo_check_if_needs_reflow(struct lvinfo_item * items[], int count,
             {
                 too_tight = 1;
             }
-            prev_right = items[i]->x + items[i]->width/2;
+
+            if (i < count) prev_right = items[i]->x + items[i]->width/2;
```
Comment (reticulatedpines) [inline src/lvinfo.c]: Let's split this into a two-line indented if, to test this workflow

**Review by reticulatedpines — CHANGES_REQUESTED** [review]
A test comment for "finish your review" part of the process

File: src/lvinfo.c (hunk 1 above)
Comment (kitor) [reply src/lvinfo.c]: Updated in 242ce8130

**Pull request merged**